src/containers/PaymentSettingsPage/PaymentSettingsPage.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { bool } from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { injectIntl, intlShape, FormattedMessage } from 'react-intl';
+import { ensureCurrentUser } from '../../util/data';
+import { propTypes } from '../../util/types';
+import { isScrollingDisabled } from '../../ducks/UI.duck';
+import {
+  LayoutSideNavigation,
+  LayoutWrapperMain,
+  LayoutWrapperSideNav,
+  LayoutWrapperTopbar,
+  LayoutWrapperFooter,
+  Footer,
+  Page,
+  UserNav,
+} from '../../components';
+import { TopbarContainer } from '../../containers';
+import { PaymentSettingsForm } from '../../forms';
+
+import css from './PaymentSettingsPage.css';
```

Comment:
The page component look good 👍 
I'd want you to make one big renaming task. In design, this page is called Payment Methods - so, it's probably better that the component, route, title, and design use the same name.

![Screenshot 2019-07-18 at 12 42 42](https://user-images.githubusercontent.com/717315/61447791-55043280-a95a-11e9-9da5-738aff40cc2b.png)